Tests/Fixtures/Generated/JSON/inline-swift5/compilation-configuration.yml

```diff
@@ -1,4 +0,0 @@
---- !ruby/object:CompilationConfiguration
-common:
-  swift_versions:
-    - 5
```

Comment:
This file should not have been deleted: it's not a fixture per se, but instead used by our `Rakefile` task `rake output:compile` – which is in charge of compiling the generated outputs to ensure that they are producing valid Swift that is able to compile

Reply:
oops